app/views/events/show.html.erb

```diff
@@ -98,7 +98,7 @@
 
 <% if user_signed_in? %>
 
-    <% hangout_is_live = @hangout.try!(:live?) %>
+    <% hangout_is_live = @recent_hangout.try!(:live?) %>
```

Comment:
The bug is caused by `@hangout` being always nil because its nowhere initialized. My guess is that it is a mistake by author which rather meant to write `@recent_hangout` and not `@hangout`.